docs/manual/guide/services/ServiceDescriptors.md

```diff
@@ -46,6 +46,10 @@ Query string parameters can also be extracted from the path, using a `&` separat
 
 @[call-query-string-parameters](code/docs/services/FirstDescriptor.java)
 
+
+When you use `call`, `namedCall` or `pathCall`, if Lagom maps that down to REST, Lagom will make a best effort attempt to map it down to REST in a semantic fashion, so that means if there is a request message, it will use `POST`, if there's none, it will use `GET`.
```

Comment:
It also looks like `GET` is _always_ used for streaming requests, even if there is no request message. @jroper can you confirm?